docker/Dockerfile

```diff
@@ -30,4 +30,4 @@ COPY --from=builder --chmod=0755 /app/target/aurcache /usr/local/bin/aurcache
 COPY --chmod=0755 docker/entrypoint.sh /entrypoint.sh
 
 WORKDIR /app
-CMD /entrypoint.sh
+CMD ["/entrypoint.sh"]
```

Comment:
These are mostly unrelated and I can move them out of this PR, they were just reporting warnings when building docker images.